channeldb/channel.go

```diff
@@ -849,6 +849,25 @@ type OpenChannel struct {
 	sync.RWMutex
 }
 
+// String returns a string representation of the channel.
+func (c *OpenChannel) String() string {
+	indexStr := "height=%v, htlc_index=%v, log_index=%v"
+
+	commit := c.LocalCommitment
+	local := fmt.Sprintf(indexStr, commit.CommitHeight,
+		commit.LocalHtlcIndex, commit.LocalLogIndex)
+
+	commit = c.RemoteCommitment
+	remote := fmt.Sprintf(indexStr, commit.CommitHeight,
+		commit.LocalHtlcIndex, commit.LocalLogIndex)
```

Comment:
meant to be `commit.RemoteHtlcIndex, commit.RemoteLogIndex)` right?

Reply:
It's addressing the remoteCommitment (just tricks the eye).

Reply:
ah indeed! thanks 🙈

Reply:
decided to populate both local/remote indexes on both local/remote commitments, and yeah it's tricky😂